components/Nav/Nav.js

```diff
@@ -27,6 +27,9 @@ function Nav(props = {}) {
 
   return (
     <Styled>
+      <CustomLink href="/discover" color="fg">
```

Comment:
@codingwithlis I think i would like for this to be to the right of the Menu Icon instead of the "Watch Now" button

Reply:
Let me know if this looks good!

<img width="498" alt="Screen Shot 2021-05-17 at 3 09 40 PM" src="https://user-images.githubusercontent.com/46769629/118543229-02485d00-b722-11eb-8882-12b7cf15c8d5.png">